framework/db/ColumnSchemaBuilder.php

```diff
@@ -223,12 +299,65 @@ protected function buildCheckString()
     }
 
     /**
-     * Builds the unsigned string for column.
+     * Builds the unsigned string for column. Defaults to unsupported.
      * @return string a string containing UNSIGNED keyword.
      * @since 2.0.7
      */
     protected function buildUnsignedString()
     {
-        return $this->isUnsigned ? ' UNSIGNED' : '';
+        return '';
+    }
+
+    /**
+     * Builds the after constraint for the column. Defaults to unsupported.
+     * @return string a string containing the AFTER constraint.
+     * @since 2.0.8
+     */
+    protected function buildAfterString()
+    {
+        return '';
+    }
+
+    /**
+     * Builds the first constraint for the column. Defaults to unsupported.
+     * @return string a string containing the FIRST constraint.
+     * @since 2.0.8
+     */
+    protected function buildFirstString()
+    {
+        return '';
+    }
+
+    /**
+     * Returns the category of the column type.
+     * @return string a string containing the column type category name.
+     * @since 2.0.8
+     */
+    protected function getTypeCategory()
+    {
+        return $this->categoryMap[$this->type];
+    }
+
+    /**
+     * Returns the complete column definition from input format
+     * @param string $format the format of the definition.
+     * @return string a string containing the complete column definition.
+     * @since 2.0.8
+     */
+    protected function buildCompleteString($format)
+    {
+        $placeholderValues = [
+            '{type}' => $this->type,
+            '{length}' => $this->buildLengthString(),
+            '{unsigned}' => $this->buildUnsignedString(),
+            '{notnull}' => $this->buildNotNullString(),
+            '{unique}' => $this->buildUniqueString(),
+            '{default}' => $this->buildDefaultString(),
+            '{check}' => $this->buildCheckString(),
+            '{pos}' => ($this->isFirst) ?
```

Comment:
Nope, in that case, if `{pos}` was present in the template it would be replaced by the the result of `buildAfterString()`, which will be an empty string.